crm_l10n_eu_nace/README.rst

```diff
@@ -0,0 +1,77 @@
+===============================
+European NACE categories in CRM
+===============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcrm-lightgray.png?logo=github
+    :target: https://github.com/OCA/crm/tree/12.0/crm_l10n_eu_nace
+    :alt: OCA/crm
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/crm-12-0/crm-12-0-crm_l10n_eu_nace
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/111/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds the concept of NACE activity to the CRM main objects (lead,
+opportunity), and propagates them through the workflow.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Bug Tracker
```

Comment:
Add `INSTALL.rst` saying that it depends on `l10n_eu_nace` module in OCA/community-data-files.

Reply:
All fixed, thanks.